core.go

```diff
@@ -0,0 +1,83 @@
+package bitbox
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/google/uuid"
+	"github.com/jmbarzee/bitbox/proc"
+)
+
+// Core offers the central functionality of BitBox.
+// Core supports basic process control and interaction.
+type Core struct {
+	m         sync.Mutex
```

Comment:
Could you use [sync.RWMutex](https://golang.org/pkg/sync/#RWMutex) here to handle concurrent reads in findProcess

Reply:
Although this sounds like it falls in to case 1 for sync.Map I still lean towards sync.RWMutex. I think its more explicit and you get to continue using the built in map type

Reply:
Great! Done!